app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
```

Comment:
I was just following a pattern I saw elsewhere in the application; is there any reason not to put this in an initializer?